README.md

```diff
@@ -52,7 +52,7 @@ git clone https://github.com/spinalcordtoolbox/PAM50-normalized-metrics.git
 
 ### `spinal_cord` directory
 
-The CSV files were generated using [process_data_spine-generic.sh, r20230222](https://github.com/sct-pipeline/dcm-metric-normalization/blob/r20230222/scripts/process_data_spine-generic.sh) script from the [spine-generic/data-multi-subject, r20230223](https://github.com/spine-generic/data-multi-subject/tree/r20230223) dataset.
-Spinal cord segmentation masks from [derivatives/labels](https://github.com/spine-generic/data-multi-subject/tree/r20230223/derivatives/labels) were used (files with the `seg-manual.nii.gz` suffix). These masks were produced by `sct_deepseg_sc` and manually corrected.
-[SCT v6.0](https://github.com/spinalcordtoolbox/spinalcordtoolbox/tree/6.0) was used to compute the morphometric measures.
+The CSV files were generated using [process_data_spine-generic.sh, r20230222](https://github.com/sct-pipeline/dcm-metric-normalization/blob/r20230222/scripts/process_data_spine-generic.sh) script from the [spine-generic/data-multi-subject, r20250314](https://github.com/spine-generic/data-multi-subject/releases/tag/r20250314) dataset.
```

Comment:
just to confirm, did you use the same script?
[process_data_spine-generic.sh, r20230222](https://github.com/sct-pipeline/dcm-metric-normalization/blob/r20230222/scripts/process_data_spine-generic.sh)

Reply:
opened a PR about it: https://github.com/sct-pipeline/dcm-metric-normalization/pull/25

Reply:
done in https://github.com/spinalcordtoolbox/PAM50-normalized-metrics/pull/23/commits/3a3f763880564bba8f5b38748641503b0649f41b